lesson_05/shawndunsmore/README.md

```diff
@@ -0,0 +1,6 @@
+## User Stories
+* As the user I would like to see recent posts iv'e liked and shared.
```

Comment:
Please review the article again that discusses the format of user stories. You are missing the purpose statement in the user stories.